src/app/main/component/places/places.component.ts

```diff
@@ -332,6 +338,14 @@ export class PlacesComponent implements OnInit, OnDestroy {
       });
   }
 
+  modifyMapContainer() {
```

Comment:
Add a return type (:void) if this method is really necessary and you can't specify this styles in the scss file instead

Reply:
add return type (the styles cannot be added to scss file couse the selector generates after the page was rendered)

Reply:
so the only ways are to make ViewEncupsulation.None(which is a bad idea) or to use the DOM to add the styles in afterViewInit

Reply:
You can achieve the same result with built-in features. Just use the width and height properties of google-map component. 
![image](https://github.com/user-attachments/assets/2db375f4-98d5-47af-a0df-522f26b12da6)


Also you can use
 `:host ::ng-deep .class {}` 
to rewrite style and save encapsulation but there is no need for this in this case.